versioned_plugins.rb

```diff
@@ -82,16 +86,17 @@ def check_rate_limit!
   end
 
   def generate_docs
-    regex = Regexp.new(plugin_regex)
     puts "writing to #{logstash_docs_path}"
     repos = octo.org_repos("logstash-plugins")
     repos = repos.map {|repo| repo.name }.select {|repo| repo.match(plugin_regex) }
     repos = (repos - PLUGIN_SKIP_LIST).sort.uniq.map {|repo| "logstash-plugins/#{repo}"}
+    repos = repos.concat(ADDITIONAL_PLUGINS)
 
     puts "found #{repos.size} repos"
 
     # TODO: make less convoluted
     timestamp_reference = since || Time.strptime($TIMESTAMP_REFERENCE, "%a, %d %b %Y %H:%M:%S %Z")
+    puts "Generating docs since #{timestamp_reference.inspect}"
```

Comment:
Let's leave debug message here, this helps to analyze some cases about why plugin doc is not generated.